website/docs/chapter-05/01_advanced_variables.md

```diff
@@ -0,0 +1,453 @@
+
+::::lo[Learning Objectives]
+
+:::K1[LO-5.1.5.2]
+
+Recall that `&{dict}` unpacks to multiple key=value pairs when accessed
+
+:::
+
+::::
+
+Variables containing dictionaries are typically accessed using the normal dollar-syntax `${var}`.
+You can also access individual values by their keys using `${var}[key]` or `${var.key}` for Robot Framework Dictionaries.
+
+However, in some cases, it is useful to unpack the key-value pairs of a dictionary variable to use them as a sequence of multiple key-value pairs. This is done using the ampersand-syntax `&{var}` when accessing the variable.
+
+Example:
+
+```robotframework
+*** Variables ***
+&{participant_one}    name=Alice    age=23
+&{participant_two}    name=Bob      age=42
+
+*** Keywords ***
+Log Participant
+    [Arguments]    ${name}    ${age}
+    Log    ${name} is ${age} years old
+
+*** Test Cases ***
+Test Dictionary Variables
+    Log Participant    John    33
+    Log Participant    name=Pekka    age=44
+    Log Participant    &{participant_one}
+    Log Participant    &{participant_two}
+```
+
+Instead of calling the keyword `Log Participant` with two arguments, it is possible to use the unpacked dictionary variables `&{participant_one}` and `&{participant_two}` to call the keyword with two named arguments.
+The dictionary keys act as the argument names and the values as the argument values.
+
+
+
+## 5.1.6 Built-In Variables
+
+::::lo[Learning Objectives]
+
+:::K1[LO-5.1.6]
+
+Recall that Robot Framework provides access to execution information via Built-In variables
+
+:::
+
+::::
+
+Robot Framework has a set of built-in variables that can be used in test cases, keywords, and other places. Some examples are:
+
+| Variable         | Description                                                                                                                                                    |
+|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `${EMPTY}`         | An empty string.                                                                                                                                               |
+| `${SPACE}`         | A single space character.                                                                                                                                      |
+| `${CURDIR}`        | An absolute path to the directory where the current suite or resource file is located. This variable is case-sensitive.                                       |
+| `${EXECDIR}`       | An absolute path to the directory where test execution was started from.                                                                                       |
+| `${OUTPUT_DIR}`    | An absolute path to the directory where output files, like `output.xml`, `log.html`, and `report.html`, are written.                                           |
```

Comment:
I would suggest to use a consistent style for all the variable names: 

- `${EXECDIR}`
- `${EXEC DIR}`
- `${EXEC_DIR}`